unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ECSRuntime/ComponentsGroup/ECSComponentsGroupOfTwo.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using DCL.Controllers;
+using DCL.Models;
+
+namespace DCL.ECSRuntime
+{
+    public class ECSComponentsGroupOfTwo<T1, T2> : IECSComponentsGroup, IECSReadOnlyComponentsGroupOfTwo<T1, T2>
```

Comment:
I think you can just name it `ECSComponentsGroup<T1, T2>` it won't conflict with, let's say, `ECSComponentsGroup<T1, T2, T3>`.

Reply:
done. you are right and I can't even recall why did I want to make it that explicit in the class name